components/src/03-modules/mini-cart/MiniCart.js

```diff
@@ -35,6 +37,27 @@ export default {
     taxAmount: {
       type: [String, Number],
       default: 0
+    },
```

Comment:
Lets move `removeButton` props below all required to keep an order from the other components :slightly_smiling_face: